cssdb.settings.json

```
@@ -205,6 +205,19 @@
       }
     ]
   },
+  {
+    "id": "css-color-function",
```

```suggestion
    "id": "color-function",
```

I think most don't have this prefix.
But I named the issue like this, so I added confusion here :)